### PR TITLE
Home: created Home Page

### PR DESCRIPTION
**Pull request by Divya70**
Home page

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,199 @@
+*{
+      margin: 0px;
+      padding: 0px;
+}
+:root {
+      --primary-color: #F94892;
+      --secondary-color: #323232;
+      --black-color: hsl(0, 0%, 0%);
```
Comment (gadgetfather) [inline index.css]: Instead of plain black, you can use the shade of black.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                        <a href="docs.html"><button class="doc-button">Documentation</button></a> 
+                        </div>
+               <div class="home-image">
+                     <img src="images/home.svg" />
+               </div>
+              
+            </div>
+            
+            <footer class="footer">
+                  <div class="footer-header">Made with <span style="color: red;">❤</span> by Divya Namdev</div>
```
Comment (gadgetfather) [inline index.html:57]: it would look good if you add space between this text and social icons.

File: index.css (hunk 1 above)
Comment (Divya70) [reply index.css]: yes sure

**Review by gadgetfather — APPROVED** [review]
design is good 👍

File: index.css [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+      --white-color: #F9F9F9;
+  }
+  body{
+        background-color: var(--black-color);
+        
+  }
+  .container{
+        /* background-color: red; */
```
Comment (VishalPatil18) [inline index.css:16]: Try to avoid including commented code in the PR.

File: index.css [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+        /* position:fixed;
+        top: 0; */
+        width: 100%;
+      
+        /* overflow: hidden; */
+
+  }
+ 
+  .nav-home-page
+  {
```
Comment (VishalPatil18) [inline index.css:36]: The formatting is a bit messed up here. Can be corrected.

File: index.css [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+        display: flex;
+        flex-direction: column;
+        margin-left: 5rem;
+        font-size: 2rem;
+        color: var(--primary-color);
+        
+  }
+  .sub-title-name{
+font-size: 1.1rem;
+color: var(--white-color);
```
Comment (VishalPatil18) [inline index.css:62]: Formatting can be improved here.

File: index.css [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+        /* color: green; */
+        padding: 2rem;
+        
+  }
+  .home{
+        color: var(--white-color);
+       padding: 2rem;
+       text-decoration: none;
+
+
```
Comment (VishalPatil18) [inline index.css:81]: Try to avoid empty space.

File: index.css [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+  .doc-button{
+       padding: 10px;
+       margin-top: 20px;
+       background-color: var(--primary-color);
+       margin-left: 6rem;
+       border-radius: 5px;
+       color: var(--white-color);
+       border: 2px solid white;
+       font-size: 1.1rem;
```
Comment (VishalPatil18) [inline index.css:127]: As this class has multiple properties you could sort them alphabetically as that helps in debugging as well as code maintenance.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+                  <div class="main-data">
+                         <h1 class="main-heading">
+                         My First Component Library
+                         </h1>
+                         <p>This is my first component library designed using HTML and CSS. This is more customizable.Do not think more about designing part, Think About logic part </p>
+                        <a href="docs.html"><button class="doc-button">Documentation</button></a> 
+                        </div>
+               <div class="home-image">
+                     <img src="images/home.svg" />
+               </div>
```
Comment (VishalPatil18) [inline index.html:52]: Code Formatting can be improved here.

**Review by Abhijeet43 — APPROVED** [review]
Code looks fine... design is good!!!

**Review by VishalPatil18 — CHANGES_REQUESTED** [review]
A few formatting suggestions could help make your code clean. Apart from that, the landing page looks quite good! Great job Divya 👍

**Pull request merged**